### PR TITLE
chore: upgrade to webpacker@4 / babel@7

### PR DESCRIPTION
**Pull request by cbarber**
This fixes regressions from the webpack-dev-server security upgrade

### CODE REVIEW

**Comment by aaddaamm** [comment]
Looks like there are some linter issues with the upgrade to eslint.

**Comment by cbarber** [comment]
@aaddaamm fixed 'em!

File: app/javascript/components/widgets/weather/hourly-temps.jsx [hunk 1]
```diff
@@ -4,7 +4,7 @@ import gql from 'graphql-tag';
 import { take } from 'ramda';
 import styled from 'styled-components';
 import { colors, fontSizes, spacing, fonts } from '../../../lib/theme';
-import { Row } from '../..//row';
+import { Row } from '../../row';
```
Comment (aaddaamm) [inline app/javascript/components/widgets/weather/hourly-temps.jsx]: 🦅 👁

**Comment by aaddaamm** [comment]
@cbarber i think this PR looks good. I did add a [ticket](https://www.pivotaltracker.com/story/show/165538921) in the backlog to update the README for non Docker instructions. Since that is outside the scope of this fix that should be a separate PR.

**Pull request merged**